### ISSUE TITLE
Training Data

### ISSUE THREAD

**Issue by xinformatics** [IdoSpringer/ERGO-II#6]
Hi Ido,

Could you please share the training data for ERGO-II? Also, is the training data for the previous ERGO model is same as ERGO-II?

Thank you,

**Comment by IdoSpringer** [comment]
Hi, ERGO-II training data is the same as ERGO training data (however we utilized more features).
There are two main databases:
McPAS - http://friedmanlab.weizmann.ac.il/McPAS-TCR/
VDJdb - https://vdjdb.cdr3.net/search
See the paper for more details.
Ido

**Comment by xinformatics** [comment]
Hi, thanks for the reply. Could you please share the negative dataset? I am trying to generate the negative set as described in the paper but unable to do so.

**Comment by IdoSpringer** [comment]
The pickle files in the samples directory include also the negative examples.
https://github.com/IdoSpringer/ERGO-II/tree/master/Samples

**Comment by xinformatics** [comment]
Thank you so much. It solved the issue. Also, could you please tell if metrics such as aur-PR, MCC were used to evaluate the model performance?

**Comment by IdoSpringer** [comment]
No, we evaluated the model performance on several tasks using (only) ROC AUC as detailed in the manuscript.

**Comment by xinformatics** [comment]
Thank you.

**Comment by xinformatics** [comment]
Hi @IdoSpringer , could you please tell if using only AUC ROC as a metric on a 1:5 imbalanced dataset is misleading? ideally for imbalanced datasets, AUC-PR and MCC are better metrics.

**Comment by IdoSpringer** [comment]
We do not believe that ROC AUC is misleading, but this is the only metric that we used. You may check other metrics.

**Comment by xinformatics** [comment]
Sure, Thank you so much for your reply.

**Comment by xinformatics** [comment]
Hi @IdoSpringer , could you tell what steps I must follow to replicate the results in the  ERGO-II paper? Thanks
